src/main/common/typeconversion.c

```diff
@@ -188,7 +188,8 @@ char *ftoa(float x, char *floatString)
 
     dpLocation = strlen(intString2) - 3;
 
-    strncpy(floatString, intString2, dpLocation);
+    // https://github.com/emuflight/EmuFlight/pull/125#issuecomment-609475459
+    memcpy(floatString, intString2, dpLocation);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5342715)